plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/StmtConversionVisitor.kt

```diff
@@ -355,6 +399,16 @@ object StmtConversionVisitor : FirVisitor<Exp, StmtConversionContext<ResultTrack
     private val FirResolvable.calleeNamedFunctionSymbol: FirNamedFunctionSymbol
         get() = calleeReference.toResolvedNamedFunctionSymbol()!!
 
+    @OptIn(SymbolInternals::class)
+    private val FirPropertyAccessorSymbol.hasBody: Boolean
+        get() = this.fir.body != null
+
+    private val FirVariableAssignment.isAssigningToClassField: Boolean
```

Comment:
I think this could be an extension property on the `lValue` instead of on the assignment.